### ISSUE TITLE
tRNA Modifications of S. pombe

### ISSUE THREAD

**Issue by dexink** [pombase/curation#3304]
Currently curating tRNA modifications and associated genes for COGs database during NIH summer internship. A possible opportunity for contributing to these annotations as well, simultaneously (two-birds, one-stone).

Modomics, a landmark database dedicated to tRNA modifications, appears to have a number of incorrectly assigned COGs for tRNA modification enzymes. My guess is that the misannotations do not stop with their database and have likely impacted others. This is in addition to the massive expansion and editing that has been necessary so far for the tRNA modifications pathway catalog of the COGs database (NCBI), which went from 58 COGs to ~102 following several iterations of the list and curating based on published data and expert consultation.

**Comment by dexink** [comment]
It is unclear whether Cbf5 of S. pombe also pseudouridinylates tRNA; published data seems to indicate that this homolog in bacteria performs this action for tRNA but only modifies rRNA/mRNA/snRNA in eukaryota.
verdict: needs more experimental evidence for annotation as a tRNA modification gene in S. pombe despite its homology to TruB in E. coli

**Comment by ValWood** [comment]
Cbf5 isn't annotated to pseudouridinylation  of tRNA?

We annotated to 
 based on the S. cerevisiae ortholog. It is fairly well characterised in yeast. The E. coli ortholog is only because they are family members, but there are multiple subfunctionalised members of the family  in higher eukaryotes
https://www.pombase.org/gene_subset/interpro:IPR020103

**Comment by dexink** [comment]
I completely agree; I was just hesitant to transfer this annotation because tRNA modification diversity varies quite a lot in between organisms, depending.
